gson/src/main/java/com/google/gson/FieldNamingStrategy.java

```diff
@@ -29,12 +31,24 @@
  */
 public interface FieldNamingStrategy {
 
-  /**
-   * Translates the field name into its JSON field name representation.
-   *
-   * @param f the field object that we are translating
-   * @return the translated field name.
-   * @since 1.3
-   */
-  public String translateName(Field f);
+    /**
+     * Translates the field name into its JSON field name representation.
+     *
+     * @param f the field object that we are translating
+     * @return the translated field name.
+     * @since 1.3
+     */
+    public String translateName(Field f);
+
+    /**
+     * Translates the field name into its alternative JSON field name representations, Gson will use the first existing
+     * JSON field name when deserialising the object. When serialising, the frist value will be used.
+     *
+     * @param f the field object that we are translating
+     * @return list of the translated field names that will be checked during deserialisation. For serialization only
+     * the first oe will be used.
```

Comment:
let's fix the typo and break the line on the sentence ;) 
```suggestion
     * @return list of the translated field names that will be checked during deserialisation.
     * For serialization only the first one will be used.
```